### DISCUSSION TITLE
layout bsp: two windows make top half/bottom half??

### DISCUSSION THREAD

**Discussion by molleweide** [asmvik/yabai#1088] [Q&A] [answered]
With layout bsp and two windows, how do I make them occupy top half and bottom half?

Is this possible?

I have been playing around a lot with ratio, window placement and grid now so I am starting to understand how things come together. But I am not sure how to put one window occupy bottom/top 50 percent. It is a lot of fun however to play with yabai.

**Comment by Liquidmantis** [comment, marked as answer]
I believe it’s in the space namespace, but there’s a rotate command that takes an arbitrary degree amount.  Just use 90 and 270 for reversing.

**Reply by molleweide** [reply]
Thanks for mentioning rotate - that was it!!
